framework_tvb/tvb/core/services/settings_service.py

```diff
@@ -124,10 +126,12 @@ def __init__(self):
             self.KEY_CLUSTER_SCHEDULER: {'label': 'Cluster Scheduler', 'readonly': False,
                                          'value': TvbProfile.current.cluster.CLUSTER_SCHEDULER, 'type': 'select',
                                          'options': TvbProfile.current.cluster.ACCEPTED_SCHEDULERS},
+            self.KEY_HPC: {'label': 'Run operations on HPC', 'value': TvbProfile.current.hpc.IS_HPC_RUN,
+                               'description': 'Check this only if the web-server machine has access to HPC.',
+                               'dtype': 'primitive', 'type': 'boolean'},
```

Comment:
We should review how much is necessary for such settings to appear in WEB GUI, or they are enough to be read from `.tvb.configuration` file only